graph/api/src/main/java/org/jboss/windup/graph/service/GraphService.java

```diff
@@ -34,6 +34,12 @@ public GraphService(GraphContext context, Class<T> type)
         this.type = type;
     }
 
+    @SuppressWarnings("unchecked")
+    public static <T extends WindupVertexFrame> T reload(GraphContext context, T frame)
```

Comment:
I'm still not sure about this name since it doesn't reload anything. Maybe "reframe"? Or maybe I am the only one who is confused by this name. :)